common/flatpak-installation.c

```diff
@@ -1112,9 +1115,16 @@ flatpak_installation_list_installed_refs_for_update (FlatpakInstallation *self,
        * This makes sure that the ref (maybe an app or runtime) remains in usable
        * state and fixes itself through an update.
        */
-      state = flatpak_dir_get_remote_state_optional (dir, remote_name, FALSE, cancellable, error);
+      state = g_hash_table_lookup (remote_states, remote_name);
       if (state == NULL)
-        continue;
+        {
+
```

Comment:
Just noticed, this extra line got in upstream too, so I will fixup! that upstream-only.

Reply:
Done [here](https://github.com/flatpak/flatpak/pull/3376/files)